test/features/css.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
```

Comment:
This file is not needed because the same checks are already performed by the monitoring `x-amples`